test/unit/exception_handling_test.rb

```diff
@@ -111,10 +111,11 @@ class SmtpClientErrbackStub < SmtpClientStub
 
     context "#log_error" do
       should "take in additional keyword args as logging context and pass them to the logger" do
-        ExceptionHandling.log_error('This is an Error', 'This is the prefix context', service_name: 'exception_handling')
+        error_id = ExceptionHandling.log_error('This is an Error', 'This is the prefix context', service_name: 'exception_handling')
```

Comment:
instead of shoehorning this into and existing test, could we create a more targeted test for the new contract? repurposing the deleted `should "return nil"` test seems like a good thing to do.

Reply:
I thought the assertion would speak for itself. I tend to think we sometimes overdo it by breaking out so many individual assertions into their own blocks. Repurposing the existing block seems fine here though and I don't feel too strongly about it. 

Done.

Reply:
This is definitely something to keep in mind, but i think if tests are slim and direct and testing single responsibility, the cognitive load of figuring out what broke given a change that was made is much more obvious.  The `should` statements should give very direct statements of what is expected to happen so that they point directly to the issue, rather than relying on the assertion of `expected x to be y` to indicate that you broke a return somewhere.